<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>storageWindow</class>
 <widget class="QWidget" name="storageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Storage</string>
  </property>
  <widget class="QListWidget" name="regexpList">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>290</width>
     <height>160</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="batchSize">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>170</y>
     <width>140</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>145</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
